packages/plugins/chainflip/src/plugin.ts

```diff
@@ -1,27 +1,29 @@
-import type { QuoteResponseRoute } from "@swapkit/api";
 import {
   AssetValue,
   type EVMWallets,
   ProviderName,
   type SubstrateWallets,
   SwapKitError,
   type SwapKitPluginParams,
-  type SwapParams,
   type UTXOWallets,
 } from "@swapkit/helpers";
 
+import type { RequestSwapDepositAddressParams } from "./types.ts";
+
 type SupportedChain = keyof (EVMWallets & SubstrateWallets & UTXOWallets);
 
 export async function confirmSwap({
   buyAsset,
   sellAsset,
   recipient,
   brokerEndpoint,
+  maxBoostFeeBps,
 }: {
   buyAsset: AssetValue;
   sellAsset: AssetValue;
   recipient: string;
   brokerEndpoint: string;
+  maxBoostFeeBps: number;
```

Comment:
It should be probably optional and defaulted to `0`

Reply:
done